### ISSUE TITLE
RuleAllTestsHaveProductionClass: Ignore JUnit specific extensions

### ISSUE THREAD

**Issue by volodya-lombrozo** [volodya-lombrozo/jtcop#193]
Quite often people place `JUnit`-specific classes right under `test` folder, I believe that `RuleAllTestsHaveProductionClass ` should accept that classes, in other words, it have to ignore them. `JUnit` classes:

```
ExecutionCondition
ParameterResolver
BeforeEachCallback
AfterEachCallback
...
etc
```

**Comment by 0pdd** [comment]
@volodya-lombrozo the puzzle [#208](https://github.com/volodya-lombrozo/jtcop/issues/208) is still not solved.

**Comment by 0pdd** [comment]
@volodya-lombrozo the puzzle [#210](https://github.com/volodya-lombrozo/jtcop/issues/210) is still not solved; solved: [#208](https://github.com/volodya-lombrozo/jtcop/issues/208).
